src/dso_api/settings.py

```diff
@@ -14,17 +14,20 @@
 from sentry_sdk.integrations.django import DjangoIntegration
 from sentry_sdk.integrations.logging import LoggingIntegration
 
+from dso_api.sentry import before_send
+
 env = environ.Env()
 
 # -- Environment
 
 BASE_DIR = Path(__file__).parents[1]
-DEBUG = env.bool("DJANGO_DEBUG", True)
+DEBUG = env.bool("DJANGO_DEBUG", False)
 
 CLOUD_ENV = env.str("CLOUD_ENV", "default")
 DJANGO_LOG_LEVEL = env.str("DJANGO_LOG_LEVEL", "INFO")
 DSO_API_LOG_LEVEL = env.str("DSO_API_LOG_LEVEL", "INFO")
 DSO_API_AUDIT_LOG_LEVEL = env.str("DSO_API_AUDIT_LOG_LEVEL", "INFO")
+SENTRY_BLOCKED_PATHS: Final[str] = env.list("SENTRY_BLOCKED_PATHS")
```

Comment:
`SENTRY_BLOCKED_PATHS: Final[str] = env.list("SENTRY_BLOCKED_PATHS", default=[])`

Reply:
Good catch! Turns out that `django-environ` does raise an exception, where I assumed that it would default to a `[]` based on the type.